DDCore/src/FieldTypes.cpp

```diff
@@ -99,16 +99,16 @@ void MultipoleField::fieldComponents(const double* pos, double* field) {
     double y2 = y*y;
     switch(coefficents.size())  {
     case 4:      // Ocupole momentum
-      by +=  coefficents[3] * (x2*x - 3.0*x*y2) + skews[3]*(y2*y - 3.0*x*y2);
```

Comment:
I think this fix is correct, but the same comment as below, shouldn't there be a 1/6 in front for normalization?